modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilderArgs.java

```diff
@@ -115,14 +126,59 @@ public void setSassCompilerClassName(String sassCompilerClassName) {
 		return s.split(",");
 	}
 
+	@Parameter(
+		description = "Whether to append the current timestamp to the URLs in the @import CSS at-rules.",
+		names = "sass.append.css.import.timestamps"
+	)
 	private boolean _appendCssImportTimestamps = APPEND_CSS_IMPORT_TIMESTAMPS;
-	private String[] _dirNames = {DIR_NAME};
-	private String _docrootDirName = DOCROOT_DIR_NAME;
+
+	@Parameter(
+		description = "The name of the directories, relative to docrootDir, which contain the SCSS files to compile. All sub-directories are searched for SCSS files as well.",
+		names = "sass.dir"
+	)
+	private List<String> _dirNames = Arrays.asList(DIR_NAME);
+
+	@Parameter(
+		description = "If the java plugin is applied: The first resources directory of the main source set (by default: src/main/resources).\nIf the war plugin is applied: project.webAppDir.\nOtherwise: null",
+		names = "sass.docroot.dir"
+	)
+	private File _docrootDir = Paths.get(".", DOCROOT_DIR_NAME).toFile();
+
+	@Parameter(
+		description = "Whether to generate source maps for easier debugging.",
+		names = "sass.generate.source.map"
+	)
 	private boolean _generateSourceMap;
+
+	@Parameter(
+		description = "The name of the sub-directories where the SCSS files are compiled to. " +
+			"For each directory that contains SCSS files, a sub-directory with this name is created. ",
+		names = "sass.output.dir"
+	)
 	private String _outputDirName = OUTPUT_DIR_NAME;
+
+	@Parameter(
+		description = "The value of the portalCommonDir property if set; otherwise portalCommonFile.",
```

Comment:
We don't have a `portalCommonDir` here, do we? This was written for the Gradle plugin, please come up with a better description 😊